### ISSUE TITLE
No Voice or Map Visible on 0.13

### ISSUE THREAD

**Issue by jethro-2018** [mapbox/mapbox-navigation-android#976]
Since upgrading to 0.13 we have no voice in our application, the street map is also not visible on the navigation view. Is there a new implementation for this? We also noticed the sound button being positioned quite oddly. 

Device: Huawei P Smart, running Android 8.0

```
    minSdkVersion 23
    targetSdkVersion 27

    implementation('com.mapbox.mapboxsdk:mapbox-android-sdk:6.1.3@aar') {
        transitive = true
    }
    implementation 'com.mapbox.mapboxsdk:mapbox-android-navigation:0.13.0'
    implementation('com.mapbox.mapboxsdk:mapbox-android-navigation-ui:0.13.0') {
        transitive = true
    }
       
```

`
    <com.mapbox.services.android.navigation.ui.v5.NavigationView
        android:id="@+id/navigationView"
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        android:screenOrientation="portrait"
        app:navigationLightTheme="@style/NavigationViewLight"
        app:navigationDarkTheme="@style/NavigationViewDark"
        />
`

`
navigationView.initialize(new OnNavigationReadyCallback() {
            @Override
            public void onNavigationReady() {
 Point origin = Point.fromLngLat(oLon, oLat);
                Point destination = Point.fromLngLat(dlon, dlat);

                Log.e("origin", origin.toString());
                Log.e("destination", destination.toString());

                String awsPoolId = null;

                boolean simulateRoute = true;

                MapboxNavigationOptions navigationOptions = MapboxNavigationOptions.builder()
                        .unitType(NavigationUnitType.TYPE_METRIC)
                        .maximumDistanceOffRoute(100)
                        .build();

                NavigationViewOptions options = NavigationViewOptions.builder()
                        .navigationListener(NavigationActivity.this)
                        .progressChangeListener(NavigationActivity.this)
                        .navigationOptions(navigationOptions)
                        .directionsProfile(DirectionsCriteria.PROFILE_DRIVING)
                        .origin(origin)
                        .destination(destination)
                        .shouldSimulateRoute(simulateRoute)
                        .build();

                navigationView.startNavigation(options);
}
`

![screenshot_20180525-133654](https://user-images.githubusercontent.com/6702147/40542567-f01e2192-6020-11e8-86e8-aa46bf4146a4.jpg)

**Comment by Guardiola31337** [comment]
Hey @jethro-2018 👋 thanks for reaching out and report your issues.

> no voice in our application

This is being tracked in https://github.com/mapbox/mapbox-navigation-android/issues/922

> sound button being positioned quite oddly

👇 👀 https://github.com/mapbox/mapbox-navigation-android/issues/859 for further information
> https://github.com/mapbox/mapbox-navigation-android/pull/938 landed (bumps ConstraintLayout to `1.1.0` version and fixes the misalignment issue). It'll be included in the next release. In the meantime, you could retest this using `0.14.0-SNAPSHOT`.

> street map is also not visible on the navigation view

Could you share what you're 👀 in your Logcat? That would be really helpful to 🕵️ into the root of the issue.

**Comment by danesfeder** [comment]
Hey @jethro-2018  can you both test your issues again with our newest release of the SDK `0.14.0` - please note - there are API breaking changes: https://github.com/mapbox/mapbox-navigation-android/releases/tag/v0.14.0

**Comment by jethro-2018** [comment]
Sorry @Guardiola31337 i didn't receive a notification about your reply, i will post my logs if i encounter any of the same issues once i have tested it with 0.14!

Thanks @danesfeder

**Comment by Guardiola31337** [comment]
Hey @jethro-2018 👋 we haven't heard from you in a bit. In the meantime we released `0.15.0` 👀 https://github.com/mapbox/mapbox-navigation-android/releases/tag/v0.15.0

Is OP still an issue for you? Could you confirm? Thanks!

**Comment by Guardiola31337** [comment]
@jethro-2018 going to go ahead and close this as OP should be fixed since `0.14.0` (we have released `0.15.0` and `0.16.0-beta.1` afterwards) - if you find the issue persists, please feel free to re-open and we will continue to dig on this issue. Thanks for bringing this to our attention!
